index.html

```diff
@@ -0,0 +1,369 @@
+            This is Kyla, an expert in interior design.
+          </div>
+          <div class="slogan-subtitle">
+            <p>Renovate your home.</p>
+            <p>Write to us and</p>
+            <p>let's get started!</p>
+          </div>
+          <button class="btn btn-slogan">RENEW YOUR HOME</button>
+          <div class="slogan-text">
+            Home, sweet home
+          </div>
+
+        </div>
+      </section>
+
+      <!-- Slogan Login -->
+      <section class="slogan-login">
+        <div class="slogan-login-info">
+          <p class="slogan-login-title">Do you want to receive news
+            of promotions and new products?</p>
+          <p class="slogan-login-subtitle">Sign up for our newsletter now</p>
+          <div class="slogan-login-action">
+            <input type="text">
+            <button class="btn btn-slogan-login">Sign Up</button>
+          </div>
+          <div class="slogan-login-bottom-text">
+            Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam
+            nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat
+            volutpat. Ut wisi enim ad minim veniam, quis nostrud exerci tation
+          </div>
+        </div>
+        <div class="slogan-login-image">
+          <img src="./images/table.png" alt="table">
+        </div>
+      </section>
+
+      <!-- New Season -->
+      <section class="section-new-season">
+        <div class="new-season-title">New Season</div>
+        <div class="new-season-content">
+          <div class="new-season-decor">
+            <div class="new-season-color">
+              <div class="new-season-green"></div>
+            </div>
+            <img src="./images/bed.png" alt="bed">
+            <div class="new-season-category">Decor</div>
+          </div>
+          <div class="new-season-bedroom">
+            <img src="./images/bedroom.png" alt="bedroom">
+            <div class="new-season-category">Bedroom</div>
+            <div class="new-season-text">
+              Elegant style in your room just in one click
+            </div>
+          </div>
+          <div class="new-season-flowers">
+            <img src="./images/flowers.png" alt="flowers">
+          </div>
+        </div>
+      </section>
+
+      <!-- Do it yourself -->
+      <section class="section-do-it">
+        <div class="do-it-info">
+          <div class="do-it-title">
+            Do it yourself
+          </div>
+          <div class="do-it-subtitle">
+            <p> Renovate your house.</p>
+            <p> We advise you</p>
+          </div>
+          <div class="slogan-text do-it-text">
+            Home, sweet home
+          </div>
+        </div>
+        <div class="do-it-paint-image">
+          <img src="./images/paint.png" alt="paint" class="do-it-paint">
+          <img src="./images/tree.png" alt="tree" class="do-it-tree">
+          <img src="./images/color.png" alt="color" class="do-it-color">
```

Comment:
tổ chức lại code cho phần này. 
Không thể có chuyện để 3 cái `<image>` liền nhau như thế này được.
Khi làm frontend phải chú ý tổ chức code làm sao cho hợp lý nhất có thể để khi bên BE ghép code không xảy ra lỗi.